src/app/services/odas/odas.py

```diff
@@ -148,6 +148,10 @@ def closeConnections(self):
     def startOdasLive(self, odasPath, micConfigPath):
 
         try:
+            if self.state == ServiceState.STOPPING:
```

Comment:
I think it's weird that starting odas makes it stop if it is stopping. I think we shouldn't be allowed to call this method if odas' state is not STOPPED (the check should be in controller) and this method should just start Odas.